### DISCUSSION TITLE
Since I updated LR it doesn't work

### DISCUSSION THREAD

**Discussion by cacinha** [rsjaffe/MIDI2LR#715] [Problems]
I can't understand the error, I've uninstalled, I've restarted, I've updated, and nothing. It doesn't recognize my desk behringer

Any help?

**Comment by antzpantz** [comment]
Try deleting C:\Users\<your username>\AppData\Roaming\MIDI2LR
and C:\Users\<your username>\AppData\Roaming\Adobe\Lightroom\Modules\MIDI2LR.lrplugin
and
reinstall.
